tests/StaticTests.js

```diff
@@ -61,6 +68,8 @@ kettle.tests["static"].testDefs = [{
     },
     sequence: [{
         func: "{packageRequest}.send"
+    }, { // Send 2nd request back-to-back to test KETTLE-57
+        func: "{packageRequest2}.send"
```

Comment:
Good observation - I've added a check in an instrumented version of the "notFoundHandler" bound to the request